substrate/frame/election-provider-multi-block/src/unsigned/miner.rs

```diff
@@ -189,11 +191,11 @@ pub trait MinerConfig {
 	///
 	/// Should equal to the onchain value set in `Verifier::Config`.
 	type MaxBackersPerWinnerFinal: Get<u32>;
-	/// Maximum number of backers, per winner, per page.
-
-	/// Maximum number of pages that we may compute.
+	/// **Maximum** number of pages that we may compute.
 	///
 	/// Must be the same as configured in the [`crate::Config`].
+	/// TODO: rename to `MaxPages` for more clarity. Won't do now to not break the miner right
+	/// before WND migration.
 	type Pages: Get<u32>;
```

Comment:
@kianenigma needed? (sorry i am treating you like `@grok` here 😆 🙈 )

Reply:
ah not really, you can remove it.

Reply:
Remove the TODO comment.